changelog.d/15418.removal

```diff
@@ -0,0 +1 @@
+Since multi-user device resync api is now available, single user counterpart is removed.
```

Comment:
While this is a removal, it is a removal of an internal-only implementation detail: please rename this to `.misc`.

```suggestion
Always use multi-user device resync replication endpoints.
```

Reply:
Updated it in recent commit.